tests/unit/Core/Checkout/Promotion/Aggregate/PromotionDiscount/PromotionDiscountEntityTest.php

```diff
@@ -0,0 +1,369 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Tests\Unit\Core\Checkout\Promotion\Aggregate\PromotionDiscount;
+
+use PHPUnit\Framework\Attributes\CoversClass;
+use PHPUnit\Framework\Attributes\DataProvider;
+use PHPUnit\Framework\Attributes\Group;
+use PHPUnit\Framework\TestCase;
+use Shopware\Core\Checkout\Promotion\Aggregate\PromotionDiscount\PromotionDiscountEntity;
+use Shopware\Core\Checkout\Promotion\Aggregate\PromotionDiscountPrice\PromotionDiscountPriceCollection;
+use Shopware\Core\Checkout\Promotion\PromotionEntity;
+use Shopware\Core\Content\Rule\RuleCollection;
+use Shopware\Core\Framework\Log\Package;
+
+/**
+ * @internal
+ */
+#[CoversClass(PromotionDiscountEntity::class)]
+#[Package('checkout')]
+class PromotionDiscountEntityTest extends TestCase
```

Comment:
We don't usually test entities due to their low complexity - the issue rather points to the promotion calculation tests, as nothing was covering this case :thinking: